addons/payment_sips/models/payment.py

```diff
@@ -184,56 +164,49 @@ def _sips_form_validate(self, data):
                 # See odoo/odoo#49160
                 date = parser.parse(date).astimezone(pytz.utc).replace(tzinfo=None)
             except:
-                # will fallback on now in the write to avoid failing to
-                # register the payment because a provider formats their 
-                # dates badly or because some local library is not behaving
-                date = False
+                # fallback on now to avoid failing to register the payment
+                # because a provider formats their dates badly or because
+                # some library is not behaving
+                date = fields.Datetime.now()
         data = {
             'acquirer_reference': data.get('transactionReference'),
-            'date': date or fields.Datetime.now(),
+            'date': date,
         }
         res = False
         if status in self._sips_valid_tx_status:
-            msg = 'Payment for tx ref: %s, got response [%s], set as done.' % \
-                  (self.reference, status)
+            msg = 'ref: %s, got valid response [%s], set as done.' % (self.reference, status)
```

Comment:
```suggestion
            msg = f'ref: {self.reference}, got valid response [{status}], set as done.'
```
Join me on the the F side of strings